cardano-node/cardano-node.cabal

```diff
@@ -168,10 +168,13 @@ library
                       , ouroboros-consensus ^>= 0.1.0.2
                       , ouroboros-consensus-byron
                       , ouroboros-consensus-cardano
+                      , ouroboros-consensus-diffusion
```

Comment:
@nfrisby @dnadales :thinking: maybe we should also have `ouroboros-consensus-api` which exports a stable api and removes the direct dependency of `lib:cardano-node` on `ouroboros-consensus` :bulb: (I have no idea how intrusive that would be).